app/src/main/java/com/opsat7373/moviestars/data/model/Movie.kt

```diff
@@ -0,0 +1,25 @@
+package com.opsat7373.moviestars.data.model
+private val moviesList : Array<String> =  arrayOf("Die hard 1" , "2012", "Star wars")
+private val moviesList2 : Array<String> =  arrayOf("Die hard 2" , "2012 2", "Star wars 2")
+
+data class Movie(val original_title : String) {
```

Comment:
keep you data structures and data sources separately.
move data class to a separate file.